### ISSUE TITLE
Problem of data distribution analysis

### ISSUE THREAD

**Issue by JasonEricZhan** [JohnM-TX/ToxicWords#10]
Data train and test have different different distribution, but overlap in some place:
(https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge/discussion/52319)

**Comment by JohnM-TX** [comment]
Yes -this is interesting. I think it turned out that way because Tilii only fitted his model on words from the train set. I wrote a comment in the post asking his thoughts.

________________________________
